### ISSUE TITLE
Remove atmark "@" in the content

### ISSUE THREAD

**Issue by AkagawaTsurunaki** [AkagawaTsurunaki/BiliChatSpider#4]
Could anyone please help me to implement this function?

I found that some tricky nodes like
``` json
{
    "username": "username1",
    "content": "@username2",
    "children": [
        {
            "username": "username2",
            "content": "",
            "children": []
        }
    ]
}
```
A node whose attribute `content` only contains `@username` should be removed along with its children.

What you need is to remove the nodes that satisfy the condition mentioned above

**Comment by PermissionDog** [comment]
I can implement this.

**Comment by AkagawaTsurunaki** [comment]
You are the assignee now. Thank you! @PermissionDog

**Comment by AkagawaTsurunaki** [comment]
I found a bug here! @PermissionDog

I collected some comments from `Xiaohongshu` and call the filter function you implemented, but it seemed that many nodes I want to retain in the tree were removed improperly. 

I've sent you an e-mail with the changed codes I used and the output `json` file to illustrate the bug.

Cloud you please fix the bug?

**Comment by PermissionDog** [comment]
> I found a bug here! @PermissionDog
> 
> I collected some comments from `Xiaohongshu` and call the filter function you implemented, but it seemed that many nodes I want to retain in the tree were removed improperly.
> 
> I've sent you an e-mail with the changed codes I used and the output `json` file to illustrate the bug.
> 
> Cloud you please fix the bug?

I have fixed it in #6 .

**Comment by AkagawaTsurunaki** [comment]
Thanks for your help! @PermissionDog 
I will close this issue if the bug is confirmed to be fixed.

**Comment by AkagawaTsurunaki** [comment]
Sorry, but there is another bug I found. @PermissionDog 
Some nodes whose content contains atmark are retained improperly. 
Please fix the bug.
Thank you.

**Comment by PermissionDog** [comment]
> Sorry, but there is another bug I found. @PermissionDog Some nodes whose content contains atmark are retained improperly. Please fix the bug. Thank you.

I'm sorry for misunderstanding your intention. Now I have fixed the problem in #7 and tested it with the following test case.

input:
```json
{
    "username": "a",
    "content": "aaa",
    "children": [
        {
            "username": "b",
            "content": "@a @c ...",
            "children": [
                {
                    "username": "c",
                    "content": "@a @d"
                }, {
                    "username": "d",
                    "content": "@a",
                    "children": [
                        {
                            "username": "g",
                            "content": "ajsas"
                        }
                    ]
                }, {
                    "username": "e",
                    "content": "11aa"
                }
            ]
        }
    ]
}
```

output:

```json
{
  "username": "a",
  "content": "aaa",
  "children": [
    {
      "username": "b",
      "content": "...",
      "children": [
        {
          "username": "e",
          "content": "11aa",
          "children": []
        }
      ]
    }
  ]
}
```

**Comment by AkagawaTsurunaki** [comment]
Thank you for your help! @PermissionDog 
There is no bug and it works well now!
